app/src/main/java/com/dimelo/sampleapp/SlidingTabFragment.java

```diff
@@ -47,13 +47,25 @@ public void onViewCreated(View view, @Nullable Bundle savedInstanceState) {
 
         // Creating The ViewPagerAdapter and Passing Fragment Manager, Titles fot the Tabs and Number Of Tabs.
         mViewPagerAdapter = new ViewPagerAdapter(getFragmentManager(), tabTitles, iconIds, selectedIconIds);
-//            mViewPagerAdapter = new ViewPagerAdapter(getFragmentManager(), tabTitles, iconIds, selectedIconIds);
 
         // Assigning ViewPager View and setting the adapter
         mViewPager = (ViewPager) view.findViewById(R.id.pager);
         mViewPager.setAdapter(mViewPagerAdapter);
         mViewPager.setOffscreenPageLimit(4);
 
+        mViewPager.addOnPageChangeListener(new ViewPager.OnPageChangeListener() {
+            @Override
+            public void onPageScrolled(int position, float positionOffset, int positionOffsetPixels) {}
+
+            @Override
+            public void onPageSelected(int position) {
+                support.rcFragmentSetUserVisibleHint(position == 2);
```

Comment:
set `mDimeloChat.setUserVisibleHint()` to `true` only when support tab has been selected